File: code/demonstration/receiver/AndroidIteration3/app/src/main/java/de/fzi/sensidl/receiver/android/iteration3/sensor/Sensor.java
package de.fzi.sensidl.receiver.android.iteration3.sensor;

import java.util.UUID;

/**
 * An object that represents a sensor on the ti sensor tag
 */
public class Sensor {

	/**
	 * The uuid for the sensor service
	 */
	private UUID sensorService;
	/**
	 * The uuid for the sensor data
	 */
	private UUID sensorData;
	/**
	 * The uuid for the sensor config
	 */
	private UUID sensorConfig;

	public Sensor(UUID sensorService, UUID sensorData, UUID sensorConfig) {
		this.sensorService = sensorService;
		this.sensorData = sensorData;
		this.sensorConfig = sensorConfig;
	}

	public UUID getSensorService() {
		return sensorService;
	}

	public UUID getSensorData() {
		return sensorData;
	}

	public UUID getSensorConfig() {
		return sensorConfig;
	}
}

File: code/demonstration/receiver/XamarinIteration3/XamarinStufe3/XamarinStufe3/Sensor.cs
using System;

namespace XamarinIteration3
{
	// <summary>
	// An object that represents a sensor on the ti sensor tag 
	// </summary>
	public class Sensor
	{
		// <summary>
		// The uuid for the sensor service
		// </summary>
		public Guid sensorService { get; }

		// <summary>
		// The uuid for the sensor data
		// </summary>
		public Guid sensorData { get; }

		// <summary>
		// The uuid for the sensor config
		// </summary>
		public Guid sensorConfig { get; }

		public Sensor (Guid sensorService, Guid sensorData, Guid sensorConfig)
		{
			this.sensorService = sensorService;
			this.sensorData = sensorData;
			this.sensorConfig = sensorConfig;
		}
	}
}
